apps/web/frontend/components/organizationMemberList/removeUserFromOrganizationButton.tsx

```diff
@@ -0,0 +1,80 @@
+import { useRef } from 'react'
+import { FaArrowRightFromBracket } from 'react-icons/fa6'
+import { useMutation } from 'urql'
+
+import { FragmentType, graphql, useFragment } from '../../generated/gql'
+
+const RemoveUserFromOrganizationButtonUserFragment = graphql(`
+  fragment RemoveUserFromOrganizationButtonUser on User {
+    id
+    name
+  }
+`)
+
+const RemoveUserFromOrganizationButtonOrganizationFragment = graphql(`
+  fragment RemoveUserFromOrganizationButtonOrganization on Organization {
+    id
+    title
+  }
+`)
+
+interface RemoveUserFromOrganizationButtonProps {
+  user: FragmentType<typeof RemoveUserFromOrganizationButtonUserFragment>
+  organization: FragmentType<typeof RemoveUserFromOrganizationButtonOrganizationFragment>
+}
+
+const OrganizationMembershipDeleteMutation = graphql(`
+  mutation OrganizationMembershipDelete($organizationId: ID!, $userId: ID!) {
+    organizationMembershipDelete(organizationId: $organizationId, userId: $userId) {
+      id
+    }
+  }
+`)
+
+export const RemoveUserFromOrganizationButton = (props: RemoveUserFromOrganizationButtonProps) => {
+  const [{ fetching }, removeUser] = useMutation(OrganizationMembershipDeleteMutation)
+  const user = useFragment(RemoveUserFromOrganizationButtonUserFragment, props.user)
+  const organization = useFragment(RemoveUserFromOrganizationButtonOrganizationFragment, props.organization)
+  const dialogReference = useRef<HTMLDialogElement>(null)
+
+  const handleRemoveClick = async () => {
+    await removeUser({
+      organizationId: organization.id,
+      userId: user.id,
+    })
+  }
+
+  return (
+    <>
+      <button
+        className="btn btn-error btn-sm"
+        onClick={() => dialogReference.current?.showModal()}
+        title="Remove user from the organization"
```

Comment:
"Remove user from the organization" plus from all projects of this organization.